### DISCUSSION TITLE
Strange behavior of the left toolbar on different screen sizes

### DISCUSSION THREAD

**Discussion by HeaTTheatR** [pradyunsg/furo#503] [Potential Issue]
The width of the left toolbar is displayed differently on different monitor screen sizes.

Diagonal 15 inches:

![unknown](https://user-images.githubusercontent.com/16930280/183505755-c8a542ad-f988-4030-a0a0-139d4a5df660.png)

Diagonal 11 inches:

![2022-08-03_07 43 56](https://user-images.githubusercontent.com/16930280/183505815-7cbf0bfa-8134-4154-b721-5f33fe445130.png)

**Comment by pradyunsg** [comment]
That’s because the right sidebar is collapsed, which means that centering the visible content gives the left sidebar more space.

Why do you find this strange?

**Reply by JErfurt** [reply]
Logo takes to much useful space! Navigate bar too small

**Reply by JErfurt** [reply]
On big screen its looks nice, but on my laptop its hard to use navigation, needs to be adaptive resizable by window size like on this screenshot.
![image](https://user-images.githubusercontent.com/86729281/183514952-aa1dde22-1078-43a7-8a75-f35d5cfcca7b.png)
